pyaedt/application/Analysis.py

```diff
@@ -177,8 +177,8 @@ def materials(self):
            Materials in the project.
 
         """
-        if not self._materials:
-            from pyaedt.modules.MaterialLib import Materials
+        if not self._materials:  # Instantiate an instance of self._materials the first time the app tries to
```

Comment:
@dcrawforAtAnsys I'd put the comment at line 181 otherwise the comment gets cut and is added to next line of code

Reply:
Yes. I was planning to do this but missed it. Thanks!

Reply:
done